### PR TITLE
add gjbasegamelayer, gmanager, gamemanager, appdelegate address

### PR DESCRIPTION
**Pull request by qwix456**

### CODE REVIEW

File: bindings/2.205/GeometryDash.bro [hunk 1]
```diff
@@ -7953,10 +7953,7 @@ class GManager : cocos2d::CCNode {
 	inline GManager() {}
 
 	virtual bool init();
-	virtual TodoReturn setup();
-	virtual TodoReturn encodeDataTo(DS_Dictionary*);
-	virtual TodoReturn dataLoaded(DS_Dictionary*);
-	virtual TodoReturn firstLoad();
```
Comment (matcool) [inline bindings/2.205/GeometryDash.bro:7959]: why were these virtuals removed

**Comment by Cvolton** [comment]
why are the addresses added to 2.205, latest gd windows is 2.204

File: bindings/2.205/GeometryDash.bro [hunk 2]
```diff
@@ -4180,7 +4180,7 @@ class GameManager : GManager {
 		return GameManager::sharedState();
 	}
 
-	static GameManager* sharedState();
+	static GameManager* sharedState() = 0x121540;
```
Comment (matcool) [inline bindings/2.205/GeometryDash.bro:4183]: epic syntax error

File: bindings/2.205/GeometryDash.bro [hunk 3]
```diff
@@ -4321,7 +4321,7 @@ class GameManager : GManager {
 	TodoReturn checkUsedIcons();
 	TodoReturn claimItemsResponse(gd::string);
 	TodoReturn clearGJLog();
-	cocos2d::ccColor3B colorForIdx(int);
+	cocos2d::ccColor3B colorForIdx(int) = 0x126090;
```
Comment (matcool) [inline bindings/2.205/GeometryDash.bro:4324]: epic syntax error

**Comment by matcool** [comment]
> why are the addresses added to 2.205, latest gd windows is 2.204

i just noticed that . lmao

**Review by matcool — CHANGES_REQUESTED** [review]
these are all 2.204 addresses

**Comment by matcool** [comment]
still on 2.205..

**Pull request closed without merging**